accounts.sh

```diff
@@ -60,20 +60,26 @@ case $1 in
         do
             echo "$user" >> /vagrant/$filename
         done
-       ;;
+        ;;
     
     backup )
-       cd /vagrant
-       tar -pcvf 2018-user-home-dir.tar.gz /home /vagrant/user-accounts.txt
-       cd -
-       ;;
+        echo "" > /vagrant/*.txt
+        normal_user=$(awk -F: '($3>=1001)&&($1!="nobody"){print $1; exit}' /etc/passwd)
+        for user in $(cat /etc/shadow | grep -A 5000 $normal_user | cut -f1-2 -d:)
```

Comment:
this give me wrong result. I would prefer awk.

Reply:
I refined `cut` and `grep`. Now it will work in every case.
 
`awk` creates some problems. It used {} and () brackets. So I am unable to assign output to variable. I could use backticks but it is not a good practice.
So I removed `awk` completely.